tests/core/tools.js

```diff
@@ -855,6 +855,14 @@
 			assert.areSame( escapedSelector, '\\30 ', 'only-one-number selector' );
 		},
 
+		'test escapeCss - has-colon selector': function() {
```

Comment:
@limingli0707 This won't verify your added code if this runs in an environment where `CSS.escape` is defined. You would have to mock `CSS.escape` to be falsey during the test so it hits your code path.

Reply:
Thanks @AdamTReineke. I just checked the test here, i think CSS.escape is not defined when the tests are run. They don't really test this part I think ```			if ( window.CSS && CSS.escape ) {
				return CSS.escape( selector );
			}```
The other test case` test escapeCss - starts-with-number selector` for example. I'm following the same. But you are right, ideally the test should cover all the cases including the `CSS.escape is defined`